docs/framework/templates/customizing.md

```diff
@@ -0,0 +1,411 @@
+# Customizing Built-in Screens
+
+!!! caution "Use extensions instead"
+
+    The methods on this page are of historical interest and may be helpful in understanding legacy code, but overriding templates and php files is strongly discouraged for any new customisations. These approaches need attention whenever you upgrade and will cause a range of problems from subtle to major if you forget about them.
+    
+    As CiviCRM evolves, the recommended ways of customizing it change.   The current preferred method is to Create an Extension to implement Hooks and use the API.  If you can't find a way to use existing hooks to accomplish what you want, discuss it on StackExchange or Chat.  It may be appropriate to add a new hook to core.
+    
+    (It is possible to override templates and php in an extension, but you face the same problems when it comes to upgrading - don't do it!)
```

Comment:
Useful extension to help with this is: https://civicrm.org/extensions/extension-file-overrides